packages/cli/src/tools/installPods.js

```diff
@@ -17,6 +17,10 @@ async function installPods({
   loader?: typeof Ora,
 }) {
   try {
+    if (!(await fs.pathExists('ios'))) {
+      return;
+    }
+
     process.chdir('ios');
 
     const hasPods = await fs.pathExists('Podfile');
```

Comment:
If [this](https://github.com/react-native-community/cli/pull/365/files#r285092791) gets implemented, likely this should also be updated.